game/camera.cpp

```diff
@@ -122,6 +125,32 @@ void Camera::setMode(Camera::Mode m) {
     dst.spin.y = pl->rotation();
   }
 
+void Camera::setMarvinMode(Camera::MarvinMode m) {
+  if(camMarvinMod==m)
+    return;
+  auto pl = Gothic::inst().player();
+  if(pl==nullptr)
+    return;
+  if(m==M_Free)
+    pl->setAnim(AnimationSolver::NoAnim);
+  if(m==M_Fixed) {
+    auto offset    = dst.target - pl->cameraBone();
```

Comment:
This math odd. 
Is it another way to write `inverse(pl->cameraBone()) * dst.target` ?

Reply:
This is supposed to calculate the vector from player to camera position. Is there a better way?

Reply:
Isn't it what I'm currently doing?  Code below would lead to same offset vector.
```
   Vec3 offset(0,0,1);
   Matrix4x4 m;
   m.identity();
   m.translate(pl->cameraBone(true));
   m.inverse();
   m.translate(dst.target);
   m.project(offset);  
   cameraOffset = offset;
```

Reply:
I mean it make sense to change it, and fetch `cameraBone` from npc as matrix, with tilt and rotation.

Reply:
Edit: Looks like `offsetAng` could do the tilt.